core-bundle/src/EventListener/DbafsMetadataSubscriber.php

```diff
@@ -57,6 +57,14 @@ public function enhanceMetadata(RetrieveDbafsMetadataEvent $event): void
             $event->set('importantPart', $importantPart);
         }
 
+        // Add subtitle information
+        if (!empty($language = $row['subtitlesLanguage'] ?? null)) {
```

Comment:
Should we add a `Subtitle` class or sth alike for better DX (like we did for the other things)?

Reply:
This is how it could look:
```php
$subtitle = $file->getExtraMetadata()['subtitle']; // Class: Subtitle

$subtitle->getType();           // string-backed enum: SubtitleType
$subtitle->getSourceLanguage(); // string
$subtitle->getTitleLanguage();  // string

// Label:
$file->getExtraMetadata()['metadata']->getDefault()->getTitle();
```

I reckon having the types be an enum and using it in `subtitlesType.options` would be a good thing anyways.

Reply:
Added in:
2b4e15d20008b07f11c617605342721a9c19ccc2 5dc65b68f4420eadf65af682fe3f1838e0caae93 94ecb9e18ca738c2181f045be2664b7990fd0a35